landing/src/components/modules/bridge-in.tsx

```
@@ -352,57 +271,103 @@ export const BridgeIn = () => {
+  }, [selectedResourceIds]);
 
-      {showRemove && (
-        <Button variant="ghost" size="icon" onClick={onRemove} className="h-8 w-8 text-red-500 hover:text-red-600">
-          ×
-        </Button>
-      )}
-    </div>
+  return (
+    <>
+      {Object.entries(selectedResourceAmounts).map(([id, amount], index) => (
+        <div className="rounded-lg p-3 border border-gold/15 shadow-lg bg-dark-brown flex gap-3 items-center">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)